openapi/thinkific/Ballerina.toml

```
@@ -0,0 +1,13 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Website & App Building/Website Builders", "Cost/Freemium"]
+org = "ballerinax"
+name = "thinkific"
+icon = "icon.png"
+distribution = "2201.0.0"
```

```suggestion
distribution = "2201.0.2"
```
Seems you have used 2201.0.2